tests/CMakeLists.txt

```diff
@@ -220,14 +220,32 @@ IF (NOT DISABLE_TESTS)
 
 	################ TESTER EXECUTABLE #################
 	# Create unit test executable (openshot-test)
+	message (STATUS "Tests enabled, test executable will be built as tests/openshot-test")
 	add_executable(openshot-test
 				   tests.cpp
 				   ${OPENSHOT_TEST_FILES} )
 
 	# Link libraries to the new executable
 	target_link_libraries(openshot-test openshot ${UNITTEST++_LIBRARY})
 
-	#################### MAKE TEST ######################
+	##### RUNNING TESTS (make os_test / make test) #####
 	# Hook up the 'make os_test' target to the 'openshot-test' executable
 	ADD_CUSTOM_TARGET(os_test COMMAND openshot-test)
-ENDIF (NOT DISABLE_TESTS)
+	list(APPEND OS_TEST_CMDS "'make os_test'")
+
+	# Also hook up 'make test', if possible
+	# This requires CMake 3.11+, where the CMP0037 policy
+	# configured to 'NEW' mode will not reserve target names
+	# unless the corresponding feature is actually used
+	if (POLICY CMP0037)
+		cmake_policy(SET CMP0037 NEW)
+	endif()
+	if (CMAKE_VERSION VERSION_GREATER 3.11)
+		message(STATUS "Cmake 3.11+ detected, enabling 'test' target")
+		add_custom_target(test COMMAND openshot-test)
+		list(APPEND OS_TEST_CMDS " or " "'make test'")
```

Comment:
This is a little dumb. I'd originally written it with OS_TEST_CMDS as a list of just the commands, which would then be formatted for output:
```cmake
list(JOIN OS_TEST_CMDS ", " t)
```
...Then I discovered that `list(JOIN)` wasn't added to CMake until 3.12. So, my carefully-constructed logic was ruined, and I kind of hacked around it with a machete. Really `OS_TEST_CMDS` shouldn't even be a list, since there's no point in doing it that way without `list(JOIN)`. Oh well.